### DISCUSSION TITLE
Several Hour Ranges

### DISCUSSION THREAD

**Discussion by junejune2010** [Extended-Thunder/send-later#335] [Q&A]
Hello, 
How can I modify BusinessHours function so that I could have several hour ranges within one day, for example, 9:00-10:00 AND 14:00-15:00 etc? Unfortunately, don't know how to code.

**Comment by jph-sendlater** [comment]
This would require some coding. The BusinessHours function would be a good starting point, but you'd need to copy it into a new dynamic function and do some tweaking.

**Comment by jikamens** [comment]
Implemented in https://github.com/Extended-Thunder/send-later/blob/main/contrib/scheduling-functions/CertainDaysAndHours.txt .
